src/pages/Map/components/ToFriendAddButton/ToFriendAddButton.module.css

```
@@ -0,0 +1,16 @@
+.Button {
+  padding: 13px 23px;
+  display: flex;
+  border-radius: 22.5px;
+  background: #f2faf5;
+  box-shadow: 0 0 4px 0 rgba(0, 0, 0, 0.25);
+
+  position: absolute;
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)